README.md

```diff
@@ -1,2 +1,11 @@
 # deployments
-deployment templates for OpenCADC software components
+
+Deployment templates and operational assets for OpenCADC software components.
+
+## Documentation
+
+The repository ships a MkDocs Material site that captures operations runbooks, release
+processes, and CI/CD guidance.
+
+- Build locally: `pip install mkdocs-material mkdocs-git-revision-date-localized-plugin && mkdocs serve`
+- Published site: https://opencadc.github.io/deployments/
```

Comment:
You can remove these changes. README.md will be auto-updated with the release process.